### ISSUE TITLE
syncnet_train loss is too high for real movie.

### ISSUE THREAD

**Issue by huangx06** [Rudrabha/Wav2Lip#432]
I have recorded a video with my phone, and tested the loss with color_syncnet_train.py by loading the pretrained model lipsync_expert.pth, it showes the loss is aroud 1.3,  not the well trained loss around 0.3?  I have tested a movie generated by wav2lip, but it still shows a too high value around 1.0.   Why is that? I tried these because I also cann't train my dataset to get a good loss around 0.3, while just pinned at just 0.69. I have cleaned my dataset by fps25, 16000hz, and using syncnet_python.

**Comment by zhui064** [comment]
> 

Did you train the lipsync_expert.pth using the LRS2 dataset. The same problem occurs that the loss stay around  0.69 and can not decline.

**Comment by zhui064** [comment]
> I have recorded a video with my phone, and tested the loss with color_syncnet_train.py by loading the pretrained model lipsync_expert.pth, it showes the loss is aroud 1.3, not the well trained loss around 0.3? I have tested a movie generated by wav2lip, but it still shows a too high value around 1.0. Why is that? I tried these because I also cann't train my dataset to get a good loss around 0.3, while just pinned at just 0.69. I have cleaned my dataset by fps25, 16000hz, and using syncnet_python.

Did you train the lipsync_expert.pth using the LRS2 dataset. The same problem occurs that the loss stay around 0.69 and can not decline.

**Comment by huangx06** [comment]
I cannot access the LRS2 dataset, I use some part of AVSPEECH,  and the loss stay around 0.69 and can not decline.

**Comment by zhui064** [comment]
> I cannot access the LRS2 dataset, I use some part of AVSPEECH, and the loss stay around 0.69 and can not decline.

I found that it maybe feasible to reduce the learning rate. When I set initial LR = 0.001, it does not work. But the loss will decline when set a lower initial LR ( 1e-4 or 1e-5 )  
I also want to download the AVSpeech dataset but it's so slow. Could you provide the dataset ?

**Comment by huangx06** [comment]
I use avspeech torrent for this url   https://hyper.ai/tracker/download?torrent=8766 , and download it by Xunlei vip.

I use the learning rate as default (1e-4), I will try a smaller lr.

**Comment by OldSixOne** [comment]
> 
我用 avspeech torrent 下载 但是解压出错了

**Comment by OldSixOne** [comment]
error: End-of-centdir-64 signature not where expected (prepended bytes?)
  (attempting to process anyway)
warning [alldata.zip]:  zipfile claims to be last disk of a multi-part archive;
  attempting to process anyway, assuming all parts have been concatenated
  together in order.  Expect "errors" and warnings...true multi-part support
  doesn't exist yet (coming soon).
warning [alldata.zip]:  838860800000 extra bytes at beginning or within zipfile
  (attempting to process anyway)
file #1:  bad zipfile offset (local header sig):  838860800004
  (attempting to re-compensate)
error: invalid zip file with overlapped components (possible zip bomb)

<img width="398" alt="屏幕截图 2023-02-01 124626" src="https://user-images.githubusercontent.com/52329210/215952930-7427f268-19a5-42bf-9f3a-0eb7a0e2be48.png">

**Comment by OldSixOne** [comment]
你方便提供一下数据集么

**Comment by huangx06** [comment]
> 你方便提供一下数据集么

我用7-zip解压的。 只要第一个data.z01有，就能解压出来，我磁盘空间不够，就下到data.z05, 也能解压出来数据的。如果每个压缩包没损坏应该没啥问题才对。       我空间不够，原始数据早删了

**Comment by OldSixOne** [comment]
好的&nbsp; 我去尝试一下




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月2日(星期四) 上午10:15
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Rudrabha/Wav2Lip] syncnet_train loss is too high for real movie. (Issue #432)





  
你方便提供一下数据集么
  
我用7-zip解压的。 只要第一个data.z01有，就能解压出来，我磁盘空间不够，就下到data.z05, 也能解压出来数据的。如果每个压缩包没损坏应该没啥问题才对。       我空间不够，原始数据早删了

**Comment by liweilalala** [comment]
> I found that it maybe feasible to reduce the learning rate. When I set initial LR = 0.001, it does not work. But the loss will decline when a lower initial LR ( 1e-4 or 1e-5 ) was setting.
> I also want to download the AVSpeech dataset but it's so slow. Could you please provide the dataset ?
I set LR=1e-4, the loss will decline. But the process costs so much time and the loss has been fluctuating. Why?
